src/components/modals/ChangePasswordModal.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+
+    <div :class="{
+      'modal': true,
+      'is-active': active
+    }">
+      <div class="modal-background" @click="$emit('cancel')" ></div>
+
+      <div class="modal-content">
+        <div class="box">
+
+          <h1 class="title">
+            {{ $t('people.change_password_for') }} {{ person.name }}
+          </h1>
+
+          <form v-on:submit.prevent>
+            <text-field
+                :label="$t('people.fields.password')"
+                ref="first-password"
+                type="password"
+                @enter="confirmClicked()"
+                v-model="form.password"
+            />
+            <text-field
+                :label="$t('people.fields.password_2')"
+                type="password"
+                @enter="confirmClicked()"
+                v-model="form.password2"
+            />
+          </form>
+
+          <div class="flexrow">
+            <button
+              :class="{
+                button: true,
+                'is-primary': true,
+                'flexrow-item': true,
+                'is-loading': isLoading
+              }"
+              :disabled="false"
+              @click="confirmClicked"
+            >
+              {{ $t('profile.change_password.button') }}
+            </button>
+            <div class="filler"></div>
+
+             <button
+                class="button is-link flexrow-item"
+                @click="$emit('cancel')"
+              >
+                {{ $t("main.cancel") }}
+              </button>
+          </div>
+
+          <div
+            class="error has-text-right mt1"
+            v-if="!isValid"
+          >
+            {{ $t('profile.change_password.unvalid') }}
+          </div>
+          <div
+            class="error has-text-right mt1"
+            v-if="isError"
+          >
+            {{ $t('profile.change_password.error') }}
+          </div>
+        </div>
+      </div>
+    </div>
+    </template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex'
+import { modalMixin } from '@/components/modals/base_modal'
+
+import TextField from '@/components/widgets/TextField'
+
+import auth from '@/lib/auth'
+
+export default {
+  name: 'change-password-modal',
+  mixins: [modalMixin],
+  props: [
```

Comment:
Please use descriptors for props like in the other components.

Reply:
It's fixed